CONTRIBUTING.md

```diff
@@ -1,5 +1,117 @@
 # Contributing to the pyOpenSci Python package template
 
+
+Contributions are welcome, and they are greatly appreciated! Every little bit
+helps, and credit will always be given.
+
+## Example Contributions
+
+You can contribute in many ways, for example:
+
+* [Report bugs](#report-bugs)
+* [Fix Bugs](#fix-bugs)
+* [Implement Features](#implement-features)
+* [Write Documentation](#write-documentation)
+* [Submit Feedback](#submit-feedback)
+
+### Report Bugs
+
+Report bugs at https://github.com//juno/issues.
+
+**If you are reporting a bug, please follow the template guidelines. The more
+detailed your report, the easier and thus faster we can help you.**
+
+### Fix Bugs
+
+Look through the GitHub issues for bugs. Anything labelled with `bug` and
+`help wanted` is open to whoever wants to implement it. When you decide to work on such
+an issue, please assign yourself to it and add a comment that you'll be working on that,
+too. If you see another issue without the `help wanted` label, just post a comment, the
+maintainers are usually happy for any support that they can get.
+
+### Implement Features
+
+Look through the GitHub issues for features. Anything labelled with
+`enhancement` and `help wanted` is open to whoever wants to implement it. As
+for [fixing bugs](#fix-bugs), please assign yourself to the issue and add a comment that
+you'll be working on that, too. If another enhancement catches your fancy, but it
+doesn't have the `help wanted` label, just post a comment, the maintainers are usually
+happy for any support that they can get.
+
+### Write Documentation
+
+juno could always use more documentation, whether as
+part of the official documentation, in docstrings, or even on the web in blog
+posts, articles, and such. Just [open an issue](https://github.com//
+juno/issues) to let us know what you will be working on
```

Comment:
Juno?

Reply:
I fixed this throughout. That was a file i took from the output contributing file created via our template. Juno was the "username" and Juno also happens to be my pup. she likes open source too. 

<img width="467" alt="Screenshot 2025-06-30 at 4 24 05 PM" src="https://github.com/user-attachments/assets/a9829bb1-a27d-41be-9417-19d47340bcb9" />